src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -56,6 +63,7 @@ const DetailsSidebar = ({
 
     return (
         <SidebarContent hasTitle={hasTitle} title={<FormattedMessage {...messages.sidebarDetailsTitle} />}>
+            {hasVersions && <SidebarVersions onClick={onVersionHistoryClick} versions={versions} />}
```

Comment:
onClick is fine for now, but may need a more descriptive name in the future if/when we pull in version switching and other on* actions.